lib/mkdex.js

```diff
@@ -713,11 +752,13 @@ function computeModel (classes) {
       .map(([index]) => {
         return [index, kAccPublic | kAccNative];
       }));
+    const instanceFields = fieldItems.map((field, index) => [index, kAccPublic]);
 
     const classData = {
       constructorMethod,
       superConstructorMethod,
       virtualMethods,
+      instanceFields,
```

Comment:
Nitpick: let's move this right before `constructorMethod`, so the order is “fields followed by methods” here as well.